core/app/[locale]/layout.tsx

```diff
@@ -113,20 +115,21 @@ export default function RootLayout({ children, params: { locale } }: Props) {
       className={[inter.variable, dm_serif_text.variable, roboto_mono.variable].join(' ')}
       lang={locale}
     >
-      <head>
-        <DraftModeScript appOrigin={process.env.MAKESWIFT_APP_ORIGIN} />
-      </head>
-      <body className="flex h-screen min-w-[375px] flex-col font-body">
-        <Notifications />
-        <NextIntlClientProvider locale={locale} messages={messages}>
-          <NuqsAdapter>
-            <MakeswiftProvider previewMode={draftMode().isEnabled}>
+      <MakeswiftProvider previewMode={draftMode().isEnabled}>
```

Comment:
Moved up `<MakeswiftProvider>` so that it wraps the whole document, including `<head>` where `<CssTheme>` is injecting the styles.